tests/python/frontend/tensorflow/test_forward.py

```diff
@@ -3520,6 +3520,23 @@ def _test_forward_softsign(shape):
     _test_forward_softsign([2, 5, 2, 5])
 
 
+def test_forward_rint():
+    """test operator rint """
+
+    def _test_forward_rint(shape):
+        tf.disable_eager_execution()
+        np_data = np.random.uniform(1, 100, size=shape).astype(np.float32)
```

Comment:
include -ve floats also for testing.

Reply:
Thank you for your comments, but I don't get your point.
What does '- ve floats' mean?

Reply:
use 
`np_data = np.random.uniform(-100, 100, size=shape).astype(np.float32)`

Reply:
thinks!

Reply:
thanks!